pyproject.toml

```diff
@@ -1,19 +1,27 @@
-[tool.poetry]
+[project]
 name = "leantool"
 version = "0.1.0"
-description = ""
-authors = ["Your Name <you@example.com>"]
+description = "LeanTool CLI and MCP server for Lean 4."
 readme = "README.md"
+requires-python = ">=3.12"
+authors = [{ name = "Alok Singh", email = "alokbeniwal@gmail.com" }]
```

Comment:
it looks like you're killing poetry and replacing it with `uv`. In that case, you should commit a `uv.lock` file. 

Also don't forget to add GSM to authors list lol.